### ISSUE TITLE
Import/export reviews have very limited space on them

### ISSUE THREAD

**Issue by TonyWildish-BH** [Barts-Life-Science/AzureTRE#117]
The airlock import/export review VMs store the data on the `C:\` drive. This only has about 15-17 GB free space by the time the machine has finished booting, which means that's the limit on what can be imported/exported.

This is probably enough for the beta, but before we get to the MVP or Production, we need this limit to be much higher. 

For the MVP, we can simply increase the default disk size for a review VM by another 100 GB, which gives a good limit.

For Production, it would be great to have the limit be much higher. This could mean creating a disk based on the size of the uploaded data itself, not a fixed disk.

**Comment by TonyWildish-BH** [comment]
P.S. We need to disable malware scanning on airlock files. Files are scanned by Defender for Cloud, for which [the file size limit is 2 GB](https://learn.microsoft.com/en-us/azure/defender-for-cloud/defender-for-storage-malware-scan#malware-scanning-in-defender-for-storage-helps-protect-storage-accounts-from-malicious-content). Anything larger is automatically failed.

**Comment by TonyWildish-BH** [comment]
After discussion, we've decided to accept the 2 GB limit so we can have malware scanning. Larger amounts of data will either have to be split into multiple requests, or better, deployed via a pipeline where we can control what goes in. So this can be parked for now.

**Comment by TonyWildish-BH** [comment]
Bioku, is there a PR for the code? I don't see one.

**Comment by BIOKU-BH** [comment]
I have not created 1... I will do that.

**Comment by TonyWildish-BH** [comment]
Please do. PR's should not that they "close #nnn", then the ticket will close automatically once the PR is merged.
